sktime_dl/deeplearning/lstm/_regressor.py

```diff
@@ -0,0 +1,95 @@
+__author__ = "Withington"
+
+from tensorflow import keras
+
+from sktime_dl.deeplearning.base.estimators import BaseDeepRegressor
+from sktime_dl.deeplearning.lstm._base import LSTMNetwork
+from sktime_dl.utils import check_and_clean_data
+
+
+class LSTMRegressor(BaseDeepRegressor, LSTMNetwork):
+    ''' Long Short-Term Memory (LSTM)
+
+    Adapted from the implementation of Brownlee, J. (2018)
+
+    https://machinelearningmastery.com/how-to-develop-lstm-models-for-time-series-forecasting/
+    '''
+
+    def __init__(self,
+                 nb_epochs=200,
+                 batch_size=16,
+                 units=[50, 50],
+                 random_seed=0,
+                 verbose=False,
+                 model_name="lstm_regressor",
+                 model_save_directory=None):
+        super().__init__(
+            model_name=model_name,
+            model_save_directory=model_save_directory)
+        LSTMNetwork.__init__(
+            self,
+            units=units,
+            random_seed=random_seed)
+        '''
+        :param nb_epochs: int, the number of epochs to train the model
+        :param batch_size: int, the number of samples per gradient update.
+        :param units: int, array of size 2, the number units in each LSTM layer
```

Comment:
In the future can open this up to allow for tuning along different axes, e.g. #layers and #units in each layer, etc. Happy with this and these numbers as defaults

Reply:
Yes. It could be done with
`units=[16, 16, 32, 16]`   to create 4 layers, etc.

Unless you want to be more explicit, as per CNN and have two args like
`nb_conv_layers=2`
`filter_sizes=[6, 12]`
?